IdentityCore/src/oauth2/aad_base/MSIDAADOauth2Factory.m

```diff
@@ -231,4 +232,16 @@ - (BOOL)fillBaseToken:(MSIDBaseToken *)baseToken
     return YES;
 }
 
+
+#pragma mark - Webview
+- (MSIDWebviewFactory *)webviewFactory
+{
+    if (!_webviewFactory)
+    {
+        _webviewFactory = [[MSIDAADWebviewFactory alloc] init];
```

Comment:
nit: do we need to do it in each subclass?

Reply:
they create different type of webview factory